ellipsis.yaml

```diff
@@ -7,7 +7,7 @@ pr_review:
   
   # If quiet mode is enabled, Ellipsis will only leave reviews when it has comments, so “Looks good to me” reviews 
   # will be skipped. This can reduce clutter.
-  quiet: false
+  quiet: true
```

Comment:
let me know if anyone has strong objections against this, but I find it to be too much span every time I get a "Looks good to me email", not to mention how long it makes a PR conversation